src/Frame.jsx

```diff
@@ -78,6 +78,10 @@ export default class Frame extends Component {
 
     const doc = this.getDoc();
 
+    if (!doc) {
```

Comment:
Did you check that change locally?

I was wondering if maybe `this._isMounted = true;` from `componentDidMount` should be moved after `if (doc && doc.readyState === 'complete') {` check. That should fix this problem on first render.

Reply:
I am unable to repeat the issue, it’s mostly an error I’m seeing in analytics occurring in Chrome only and on both Windows and Mac machines. However the overall environment is out of my control (we inject the iframe into their hosted pages). 

So I can’t confirm if it’s an issue with first render all I know is that the error is “defaultView” of null and points to that portion of code.

You may have a better idea of how to fix the issue, but this will mostly be a “see if the error stops and if anyone reports negative results”. I’m willing to try an alternate change before merging this into the PR.

Reply:
I've looked into our analytics as well and it looks like it might occur also on next renders. 
Which is interesting because one reason which I know that might result in contentDocument being null is that the iframe is not yet ready.
On the second thought I think it might be connected with

```
getDoc() {
    return this.node.contentDocument; // eslint-disable-line
  }
```
I think `this.node` should also be checked.

Reply:
Hey, I've managed to create minimal reproducible example https://codesandbox.io/s/react-frame-component-defaultview-error-23001

Reply:
This actually makes perfect sense -- if the iframe is removed from the dom, the node still exists (`this.node.contentDocument` won't fail), but the contents of the iframe are unloaded so `contentDocument` returns undefined, and because react didn't unmount the node, `componentWillUnmount` doesn't fire.

Similar to your example above, if you remove the root node where the react app is mounted and then force an update, you get the same error.
```js
const rootElement = document.getElementById('root')
ReactDOM.render(<App />, rootElement)
rootElement.remove()
ReactDOM.render(<App />, rootElement)
```

Reply:
Exactly 👍
Now let's hope @ryanseddon comes around and is willing to merge 😄

Reply:
I've added a check for `this.node` as suggested by @dominiczaq .
Exciting to know that it wasn't a completely bizarre error, and it's repeatable.

Thanks all